app/src/main/java/com/p2p/meshify/ui/components/MeshifyKit.kt

```
@@ -70,642 +65,179 @@ class MorphPolygonShape(
-            cachedProgress = progress
-            cachedRotation = rotationAngle
+            cachedOutline = outline; cachedProgress = progress; cachedRotation = rotationAngle
             outline
-        } catch (e: Exception) {
-            if (!hasLoggedError) {
-                Logger.e("MorphPolygonShape -> Path calculation FAILED: ${e.message}")
-                hasLoggedError = true
-            }
-            CircleShape.createOutline(size, layoutDirection, density)
-        }
+        } catch (e: Exception) { CircleShape.createOutline(size, layoutDirection, density) }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)